scss/common/_z-layers.scss

```diff
@@ -0,0 +1,17 @@
+@use "sass:map";
+
+$z-layers: (
+	layer_null: 0,
+	layer_base: 10,
+	layer_middle: 100,
+	layer_overlay: 150,
+	layer_top: 500,
+	layer_negative_highest: -1,
+	layer_negative_middle: -100,
```

Comment:
why do you need -100 and -200?
when you set z-index -1, you already can't interact with it, why do you think you need -100 and -200 ?

Reply:
It appears to me, I saw thee such negative variables in your example.

Reply:
Done.